pkg/controller/status/status.go

```diff
@@ -182,7 +182,7 @@ func (m *statusManager) updateStatus() {
 		}
 
 		if m.IsDegraded() {
-			m.setDegraded(m.degradedReason(), m.degradedMessage())
+			m.setDegraded(string(operator.PodFailure), fmt.Sprintf("%s - %s", m.degradedReason(), m.degradedMessage()))
```

Comment:
I think understand why you did this but I don't think putting `PodFailure` here will be correct most of the time.